sources/core/Stride.Core.CompilerServices/Analyzers/TestAnalyzer.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Stride.Core.CompilerServices.Analyzers
+{
+    // PLACEHOLDER ANALYZER
```

Comment:
i would remove it as my PR has allready analyzers in it to reference for others and its proven that the base PR works.
So an example wouldnt be necessary

im fine with the rest of the PR

Reply:
Sure, I'll remove it